src/components/Search/utils.ts

```diff
@@ -184,17 +186,24 @@ export const getSearchModalStyles = (): SystemStyleObject => ({
     display: "grid",
     placeItems: "center",
     borderRadius: "base",
+    mr: "unset",
+    me: 1.5
   },
 
   ".DocSearch-Logo": {
     ".DocSearch-Label": {
-      color: "body.light",
+      color: "body.medium",
```

Comment:
Sorry to make design adjustments in this PR, but this was producing an unreadable contrast ratio:
before:
<img width="199" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/f4f0b8a4-d250-43a4-97f1-07c6438d61ae">

with change:
<img width="208" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/eb4d832d-0ee9-4cf2-a315-41ae54d3d1bb">

cc: @nloureiro if you have any other preference for this, but figured it was worth patching